scripts/release-status.mjs

```diff
@@ -0,0 +1,215 @@
+import 'dotenv/config'
+import fetch from 'node-fetch';
+import querystring from 'querystring';
+import { Octokit } from "@octokit/rest";
+
+const ptToken = process.env.PT_TOKEN;
+const ghToken = process.env.GITHUB_TOKEN;
+
+// TODO: add help text and validate command line arguments
```

Comment:
Is this a todo for sometime in the future or for this PR?

Reply:
Something for the future. I'm not sure how useful this will be yet.